src/streaming/MediaPlayerEvents.js

```diff
@@ -140,6 +140,12 @@ class MediaPlayerEvents extends EventsBase {
          */
         this.QUALITY_CHANGE_RENDERED = 'qualityChangeRendered';
 
+        /**
+         * Triggered when the new track is being rendered on-screen.
```

Comment:
Just a small correction. Given there are also audio track changes I would remove the "on-screen" part of the event description